joomla/joomla-cms
# [4.0] - Update fontawesome 4- to fontawesome 5 classes

Pull Request for Issue #27655 

### Summary of Changes
replaces fontawesome 4 classes with fontawesome 5 equivalents.

### Testing Instructions
Apply PR
run npm i
delete configuration.php file
reinstall
Browse frontend and backend to make sure no visible changes with the icons.

### Expected result
icons display as previous.

### Actual result



### Documentation Changes Required
none.
